README.md

```diff
@@ -12,8 +12,9 @@ Building requires a Linux environment, Windows users may use WSL2 without issues
 
 1. Clone the repository **with submodules**.
 2. Install **make**, **xorriso**, **bsdtar** and **rsync**.
-3. Run `make userspace-base` on the repository's root to build a base system. Run `make userspace-full` on the repository's root to build all the packages ported.
-4. Run `make` to generate an ISO image.
+3. Run `make jinx` to download jinx (the tool needed for compiling)
```

Comment:
3. Run `make jinx` to download jinx (the tool needed for compiling)
could be written as
3. Run `make jinx` to download the jinx script used for building Polaris' userspace